### ISSUE TITLE
Windows 11 x64 23H2 Blank screen

### ISSUE THREAD

**Issue by mhy-beta** [electron/fiddle#1614]
1. Blank screen
2. No response on menu item like 

> prefrences
> publish to gist
> tasks -> all items
> show me -> electron APIs -> all items
> Help -> show welcome tour

Screen Record GIF file: 
![electron-fiddle](https://github.com/user-attachments/assets/6d44e388-4a1a-48b3-a40b-e4154413d5f6)
Ref: #1357

**Comment by yangannyx** [comment]
Hey @mhy-beta can you post a screenshot of the DevTools Elements tab?

**Comment by mhy-beta** [comment]
Hi @yangannyx ,
![fiddle](https://github.com/user-attachments/assets/3658484e-6163-4c83-9431-28d0ba1904b6)

**Comment by erickzhao** [comment]
Do you get the same error in the `Console` tab in DevTools as described in https://github.com/electron/fiddle/issues/1357#issuecomment-1858088003?

**Comment by mhy-beta** [comment]
No, there is no error on the console screen. That's why I opened it as a new entry.

What is written in the console
State: Switching to Electron 30.0.2.       index.js:2
State: Downloading Electron 30.0.2.      index.js:2
Themes: getTheme() loading default     index.js:2
Fetched 2407 new Electron.                   index.js:2

**Comment by mhy-beta** [comment]
I just tried it on MacOS.

I downloaded and installed Fiddle and ran it.
Just like on Windows 11, it did not respond at all.

Also, when I wanted to move the window, it did not move at all.

I clicked Menu -> Help -> Toggle Developer Tools. The DevTools panel opened after 4-5 seconds. Then I was able to move the window.

Just like on Windows 11, there were only 4 lines in the Console with the text "Stete: Switching to Electron 30.0.2. index.js:2".
![Screenshot 2024-07-13 at 13 32 56](https://github.com/user-attachments/assets/bb9b7b23-dac6-484b-b486-7ab0332f7918)

After 1-2 minutes, it started working.
![Screenshot 2024-07-13 at 13 42 51](https://github.com/user-attachments/assets/ba6a3fb7-eac7-465e-81d5-fca406421564)

I think the problem is for MacOS Sonoma. When Fiddle first starts, it downloads something and since the downloading process is not finished, the interface is not usable. When the downloading process is finished, the application becomes operational. It downloads something or does something else, but since it does not notify the user, it seems as if the application is frozen. Since the user is not shown the operation with a progress bar, I cannot tell what it is doing. This does not seem to work to me.

After opening the app fully, it works completely normally on the second or third opening.
![Screenshot 2024-07-13 at 13 49 45](https://github.com/user-attachments/assets/091bca4d-a2b3-4605-9ab4-da06fdd0e38e)

The same situation that occurred on Windows 11 has been fixed for MacOS. A process is performed during the first run and since it does not inform the user, it appears that the application is locked and not working.

I will try again on Windows today. However, from what I see, it is the same. However, I waited for 5-6 minutes on Windows and closed it because it still did not work. I will try again and give feedback.

**Comment by mhy-beta** [comment]
The topic has come to another point.

I opened an "issue" here because I had the problem and found the solution myself, because no one helped. (funny)

If I have to explain the solution in a short way: "**Just wait.**"

**As a long answer:** "It will take a long time because Fiddle downloads and installs the file that should have been installed during the installation from the internet without giving any information in the background."

When Fiddle is installed, version 30.0.2 is displayed as "downloaded" by default in the "Electron Settings" section.
![fiddle-versions](https://github.com/user-attachments/assets/2d260aab-5dcb-457d-9bc5-60d7bc2924b3)


When I checked the "Squirrel-CheckForUpdate" log file in the installation directory, I saw that it was trying to download a file, just like in MacOS.
![fiddle-log](https://github.com/user-attachments/assets/74f3d9fa-8916-47a0-8956-f33399c721d8)


There is an "app-0.36.3" folder in the installation directory. There is also an "electron-fiddle-0.36.3-full.nupkg" file in the "packages" folder.

In other words, "Fiddle" is not responding because it is trying to download this file during startup. Because this file, which should be there during installation, does not exist, it tries to download this file at the first run time. However, it does not tell the user that it is downloading a file. According to the log file, the process that started with the date time [13.07.24 22:45:10] (I think it takes a long time because of my internet speed) ended at the date time [13.07.24 22:58:31].

**Suggestion**: @erickzhao 
Nobody opens the application, downloads a file like me and waits for it to open for 13 minutes.

Therefore, if an update is to be made on the opening screen, it asks the user and if you get approval, you start downloading.

This is a deficiency and should be added to Fiddle.

I don't think anyone's internet is super fast.
